components/buttons/buttons.css

```diff
@@ -0,0 +1,49 @@
+.btn{
+    width: max-content; 
+    letter-spacing: 1px;
+    background-color: #1FABD7;
+    padding: 0.8rem 0.8rem;
```

Comment:
padding can go to seperate files as variables.

Reply:
Okay 👍🏻